### ISSUE TITLE
any thoughts on using this without a user model?

### ISSUE THREAD

**Issue by edla** [edgarjs/ajaxful-rating#39]
we have a site that we can't require logins for, but needs to have ratings.  it would be fine if every rating was cookie based or something, or even if was completely open (anyone can put in a rating, as many times as they wanted to).

**Comment by edgarjs** [comment]
That's a good feature. I'll implement it in future versions. Thank you for your feedback

**Comment by sennor** [comment]
Is it possible to create it with the current version now?

**Comment by edgarjs** [comment]
Not really, unless you stub somehow a guest user.

**Comment by sgreenfield** [comment]
I agree, this would be a very nice feature.  It would be cool if it allowed one vote per item per session, and if the user had cookies disabled, one vote per item per IP address.

**Comment by romanenko** [comment]
It would be great!

**Comment by nilsine** [comment]
You can easily do that with devise and guest user creation based on session or cookie:
https://github.com/plataformatec/devise/wiki/How-To:-Create-a-guest-user

But be careful, I just pass long time with user and current_user helper issues. To allow rating for your guest you have to override :current_user in the options:

```
<%= ratings_for @current_or_guest_user, :current_user => @current_or_guest_user %>
```

Maybe the doc should be clearer on the distinction between user and :current_user ?
